src/lava/lib/peripherals/realsense/realsense.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+import os
+os.environ["OPENCV_IO_ENABLE_OPENEXR"] = "1"
+import cv2  # noqa: E402
```

Comment:
It looks like you may be inserting `# noqa: E402` on every subsequent import. Is this because of line 7, which is not an import? Can this be moved below all imports instead?

Reply:
The line `os.environ["OPENCV_IO_ENABLE_OPENEXR"] = "1"` has to be executed strictly before `import cv2` for this to work.
I could move this below all imports, but I am afraid we still need the `# noqa: E402` with `import cv2`